pages/task/read/views/models/project.jsx

```diff
@@ -81,20 +81,6 @@ export default function Project({ task, schema }) {
           {
             task.status === 'with-inspectorate' && (
               <Fragment>
-                <p><strong><Snippet>declarations.pel-holder.question</Snippet></strong></p>
-                <p>{declarations.authority}</p>
-                {
-                  declarationConfirmed(declarations.authority) &&
-                    <Fragment>
-                      <p>
-                        <Snippet>declarations.pel-holder.name</Snippet> {declarations['authority-pelholder-name']}
-                      </p>
-                      <p>
-                        <Snippet>declarations.pel-holder.endorsement-date</Snippet> {declarations['authority-endorsement-date']}
-                      </p>
-                    </Fragment>
-                }
```

Comment:
Can we leave this for now, for backwards compatibility reasons?

Reply:
Yes, yes we can